assets/js/translator.js

```diff
@@ -648,9 +675,22 @@ function populateTranslationList() {
 
         srcLangs = srcLangs.sort(compareLangCodes);
         dstLangs = dstLangs.sort(function (a, b) {
-            var aPossible = pairs[curSrcLang] && pairs[curSrcLang].indexOf(a) !== -1;
-            var bPossible = pairs[curSrcLang] && pairs[curSrcLang].indexOf(b) !== -1;
-
+            function determineIfPossible(a) {
+                var aVariant = a.split('_');
+                if(pairs[curSrcLang] && pairs[curSrcLang].indexOf(a) !== -1) {
```

Comment:
Also this should just be a huge return with each of these things as parenthesized lines in a conjunction :)